e2e.bats

```diff
@@ -30,3 +30,31 @@
   [ $(expr "$output" : '.*code.*401') -ne 0 ]
 }
 
+@test "accept request using OCI capability to fetch image manifest and config" {
```

Comment:
Regarding capabilities testing, we have decided not to use the e2e for the moment.

The rationale is that we are only interested that the CEL expression results in the right callback. Testing against a real registry could slow down the CI and in the end,  and also this is redundant as we are basically testing that the SDK works.
Let's simply add a test case here, instead: https://github.com/kubewarden/cel-policy/blob/ae8cee4ba626ecd092eaf3fdda8361d11e23db42/internal/cel/library/oci_test.go#L41